engine/dlib/src/test/test_intersection.cpp

```diff
@@ -105,6 +105,61 @@ TEST(dmVMath, TestFrustumSphere)
 
 }
 
+TEST(dmVMath, TestFrustumOBB)
+{
+    dmVMath::Matrix4 proj = dmVMath::Matrix4::orthographic(0.0f, FRUSTUM_WIDTH, 0.0f, FRUSTUM_HEIGHT, FRUSTUM_NEAR, FRUSTUM_FAR);
```

Comment:
I would also like to see tests with a non orthographics projection matrix, as that's what most users will use with this function.

You can ofc choose constants that make it easier to reason about in the tests.

Reply:
Ok, i'll add another test for perspective projection. 

I'm not sure wdym by "constants" though.

Reply:
Done.